low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,23 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
+      - hare-consul-agent.service
+      - hare-consul-agent-c1.service
+      - hare-consul-agent-c2.service
+      - elasticsearch.service
+      - kibana.service
+      - statsd.service
+      - rsyslog.service
+      - haproxy.service
+      - slapd.service
+      - corosync.service
```

Comment:
Could you please remove corosync, pacemaker and pcsd from this list. It will be taken care by coroync/pacemaker automatically.

Reply:
removed corosync, pacemaker and pcsd.